app/views/users/show.html.erb

```diff
@@ -0,0 +1,13 @@
+<div>
+  <h2> <%= @user.name %> </h2>
+  <h2> <%= @user.bio %> </h2>
+  <h2> <%= @user.posts_counter %> </h2>
+  <h2> <%= @user.id %> </h2>
+  <%@posts.each_with_index do |post, index|%>
+    <h4>post #<%= index + 1%></h4>
+    <h2>Title: <%=post.title%></h2>
+    <p>Description: <%=post.text%></p>
+    <p>id: <%=post.id%></p>
+  <%end%>
+  
```

Comment:
- [ ] Please make sure to add a button `View all posts` at the bottom and clicking on the button opens the user's post index page.